payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressTextFieldUI.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.annotation.RestrictTo
+import androidx.compose.foundation.clickable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.ImeAction
+
+@Composable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun AddressTextFieldUI(
+    controller: AddressTextFieldController
+) {
+    TextField(
+        textFieldController = controller,
+        imeAction = ImeAction.Next,
+        enabled = false,
+        modifier = Modifier.clickable {
```

Comment:
I wonder how hard it would be to test if this click listener is fired when clicked?

Reply:
Added a test